driver/web/rest/adminapis.go

```diff
@@ -410,39 +410,170 @@ func (h *AdminApisHandler) DeleteGroupPost(clientID string, current *model.User,
+// @ID AdminUpdateManagedGroupConfig
+// @Tags Admin
+// @Accept plain
+// @Param data body  model.ManagedGroupConfig true "body data"
+// @Param APP header string true "APP"
+// @Param id path string true "ID"
+// @Success 200
+// @Security AppUserAuth
+// @Router /api/admin/managed-group-configs [put]
+func (h *AdminApisHandler) UpdateManagedGroupConfig(clientID string, current *model.User, w http.ResponseWriter, r *http.Request) {
+	data, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		log.Printf("Error reading body on create managed group config - %s\n", err.Error())
+		http.Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest)
+		return
+	}
+
+	var config model.ManagedGroupConfig
+	err = json.Unmarshal(data, &config)
+	if err != nil {
+		log.Printf("Error on unmarshal the managed group config data - %s\n", err.Error())
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	config.ClientID = clientID
+	err = h.app.Services.UpdateManagedGroupConfig(config)
+	if err != nil {
+		log.Println(err.Error())
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "text/plain")
+	w.WriteHeader(http.StatusOK)
+	w.Write([]byte("Successfully updated"))
+}
+
+// DeleteManagedGroupConfig Deletes a managed group config
+// @Description Deletes a managed group config
+// @ID AdminDeleteManagedGroupConfig
+// @Tags Admin
+// @Accept  json
+// @Param APP header string true "APP"
+// @Param id path string true "ID"
+// @Success 200
+// @Security AppUserAuth
+// @Router /api/admin/managed-group-configs/{id} [delete]
+func (h *AdminApisHandler) DeleteManagedGroupConfig(clientID string, current *model.User, w http.ResponseWriter, r *http.Request) {
+	params := mux.Vars(r)
+
+	id := params["id"]
+	if len(id) <= 0 {
+		log.Println("id param is required")
+		http.Error(w, "id param is required", http.StatusBadRequest)
+		return
+	}
+
+	err := h.app.Services.DeleteManagedGroupConfig(id, clientID)
+	if err != nil {
+		log.Printf("error deleting managed group config for id (%s) - %s", id, err.Error())
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "text/plain")
+	w.WriteHeader(http.StatusOK)
+	w.Write([]byte("Successfully deleted"))
+}
+
+//SynchronizeAuthman Synchronizes Authman groups membership
+// @Description Synchronizes Authman groups membership
+// @Tags Admin
+// @ID AdminSynchronizeAuthman
+// @Accept json
+// @Success 200
+// @Security AppUserAuth
+// @Router /admin/authman/synchronize [post]
+func (h *AdminApisHandler) SynchronizeAuthman(clientID string, current *model.User, w http.ResponseWriter, r *http.Request) {
+	err := h.app.Services.SynchronizeAuthman(clientID)
+	if err != nil {
+		log.Printf("Error during Authman synchronization: %s", err)
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "text/plain")
```

Comment:
I would prefer to support application/json as a main content-type everywhere as a standard (it's more simpler and predictable from 3rd party integrator point of view). Respectively omit this header if the response code is just 200 with an empty body. Same as below.

CC @petyos @stefanvit

Reply:
Hi @mdryankov, I see your point, however for an empty response body I am not sure a content type header really needs to be specified at all. I will change this to the way you described, but let me know what you think.